app/Http/Controllers/PayloadController.php

```diff
@@ -75,8 +75,13 @@ public function store(PayloadCreateRequest $request, $webhookId)
             ]);
 
             return $payload->id;
-        } catch (QueryException $exception) {
+        } catch (Exception $exception) {
             DB::rollBack();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create failed',
+                'message' => 'Create failed. Something went wrong',
+            ]);
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 84</br>Blank line found at end of control structure